extension/llm/export/config/llm_config.py

```diff
@@ -90,6 +96,8 @@ class BaseConfig:
     params: Optional[str] = None
     checkpoint: Optional[str] = None
     checkpoint_dir: Optional[str] = None
+    adapter_checkpoint: Optional[str] = None
```

Comment:
can you name these the same as their corresponding foundational weight fields? I guess thats params and checkpoint? So call it lora_params and lora_checkpoint?